test/__init__.py

```diff
@@ -0,0 +1 @@
+"""Unit tests"""
```

Comment:
## Incorporate QC checks
The checks added in #385 should be incorporated here as tests as part of this PR ideally.